src/exercises/1/index.solution.ts

```diff
@@ -102,6 +102,7 @@ export function logPerson(user: User) {
 console.log('Users:');
 users.forEach(logPerson);
 
+console.log("This is a really important addition to this file!")
```

Comment:
You can make comments on single lines, or spans of lines, and you can also suggest changes.
```suggestion
console.log("This is a somewhat important addition to this file!")
```